packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botConfig.Result.v2.bicep

```diff
@@ -7,7 +7,6 @@ param provisionOutputs object
 param currentAppSettings object
 
 var botWebAppName = split(provisionOutputs.botOutput.value.webAppResourceId, '/')[8]
-
 var m365ClientId = provisionParameters['m365ClientId']
```

Comment:
this file name contains "Result", not "result.
Update it to keep consistant

Reply:
updated.